--- apparmor-profiles-2.1.2/apparmor.d/usr.sbin.avahi-daemon~	2008-02-15 03:44:35.000000000 -0200
+++ apparmor-profiles-2.1.2/apparmor.d/usr.sbin.avahi-daemon	2008-03-06 18:08:17.000000000 -0300
@@ -17,6 +17,7 @@
   /etc/avahi/hosts r,
   /etc/avahi/services/ r,
   /etc/avahi/services/*.service r,
+  /etc/lsb-release r,
   /proc/*/fd/ r,
   /usr/sbin/avahi-daemon mr,
   /usr/share/avahi/introspection/*.introspect r,
